pkg/option/config.go

```diff
@@ -957,7 +961,7 @@ var HelpFlagSections = []FlagsSection{
 			IdentityAllocationMode,
 			IdentityChangeGracePeriod,
 			FixedIdentityMapping,
-			CertsDirectory,
```

Comment:
CertsDirectory is specifically for Certificates used in L7 policy enforcement, so I think it makes sense for this to remain under policy.

Reply:
Ok, I can move it back, since I was grouping all path related info together, I moved it.

Reply:
Done.

Reply:
Done.